widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -112,26 +127,54 @@ class TaskList extends React.Component {
       });
       callback();
     } catch (error) {
-      this.handleError(error);
+      this.handleError(error, true);
       this.setState({ loading: false });
     }
   };
 
+  openDiagram = row => async () => {
+    const { connection } = this.state;
+
+    if (row.processId && row.id) {
+      const process = `${row.processId}@${row.id.split('@')[1]}`;
+      this.setState({ loading: true });
+      try {
+        const diagram = await getDiagram(connection, process);
+        this.setState({
+          diagramModal: {
+            title: `Process: ${process}`,
```

Comment:
is this possible to use i18n in that title string there?

Reply:
fixed, thanks for pointing that!